### PR TITLE
#2121 moving create unit functionality to admin tools

### LINKED ISSUE #2121: BOM - Move Create Units to Admin Tools
### Description

Right now, we allow users to create units in the material form on the bill of materials page. However, after having the feature in use for a while, it seems that there are few enough units that it makes more sense to move this feature to the admin tools page, to ensure no extraneous units are created. The units admin tool should be in the bill of materials config section on the admin tools page. It should look the same as the other ones. The delete feature is being worked on in #2084 which will end up having a lot of similar code to this one, so I'd check in with whoever is doing that ticket if it hasn't been merged

### Acceptance Criteria

- [ ] Create Unit functionality has been moved to the admin tools page

### Proposed Solution

Make sure to remove the old create units functionality in the dropdown of the units select on the material form. Look at other examples on the admin tools page to see how to set this feature up

### PR DESCRIPTION
**Pull request by Zwendle**
## Changes

Added the ability to create a new unit to the admin tools page under the Build of Materials Config section. Also removed the create unit functionality from the dropdown of the units select on the material form.

## Screenshots

Screenshot 2024-03-13 at 16 38 27
Screenshot 2024-03-13 at 16 38 55
Screenshot 2024-03-13 at 16 39 21
Screenshot 2024-03-13 at 16 39 55

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #2121

### CODE REVIEW

File: src/frontend/src/pages/AdminToolsPage/AdminToolsBOMConfig.tsx [hunk 1]
```diff
@@ -13,6 +14,9 @@ const AdminToolsBOMConfig: React.FC = () => {
         <Grid item direction="column" alignSelf="right" xs={12} md={6}>
           <MaterialTypeTable />
         </Grid>
+        <Grid item direction="column" xs={11} md={6}>
```
Comment (martin0he) [inline src/frontend/src/pages/AdminToolsPage/AdminToolsBOMConfig.tsx:17]: make this xs=12

**Review by martin0he — CHANGES_REQUESTED** [review]
great work so far this is a really good start. what you have to do now is 1) remove the dropdown option to create a new unit in the bom page (where this feature used to be, get rid of it there so that it only shows the available existing units only), 2) modify the table to just show one column with units, no need for dates, you're prob gonna have to create your own table component that's similar to the existing one just with one column and not column[] and that's more or less it for now. really great job so far !

**Review by martin0he — APPROVED** [review]
Looks good to me, good work ill get another review

**Review by martin0he — CHANGES_REQUESTED** [review]
update your PR, get rid of unnecessary notes and also update your screenshots

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**